### DISCUSSION TITLE
Do you need to apply baseline migrations to all databases?

### DISCUSSION THREAD

**Discussion by katangafor** [prisma/prisma#21532] [Q&A] [answered]
### Question

If I baseline my development database and mark it as "applied", it creates an entry in the _prisma_migrations table. So should I run that same `npx prisma migrate resolve --applied 0_init` command for every database? Including the other development databases, and production?

**Comment by janpio** [comment, marked as answer]
Generally yes, all databases should be in the same state. So depending on where and why you need to run `migrate resolve --applied`, you need to run this for all databases you are using.

**Reply by katangafor** [reply]
Ok gotchya, thanks.

So if somebody directly makes a change to the production database (on a different team, who is not using prisma), how should I account for that in prisma? Generate a migration using `migrate diff`, and then mark is as applied on all databases?

**Reply by katangafor** [reply]
Ok I've just now seen [this page on hotfixes](https://www.prisma.io/docs/guides/migrate/patching-production), which answers my last question.

**Reply by janpio** [reply]
To make it explicit: Yes. All the local databases and their `_prisma_migrations` need to account for that. Can cause quite a bit of additional work - but no way around it :/

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
